src/java.base/share/classes/java/util/Properties.java

```diff
@@ -833,6 +845,15 @@ public void save(OutputStream out, String comments)  {
      * After the entries have been written, the output stream is flushed.
      * The output stream remains open after this method returns.
      *
+     * @implNote When writing the date comment, this method checks whether the
+     * {@systemProperty java.util.Properties.storeDate} system property is set.
+     * If it is set and its value is not {@link String#isBlank() blank},
+     * then that value is written as a comment instead of writing the
+     * current date and time. Although this method doesn't mandate it, conventionally,
+     * the value of this system property represents a formatted
+     * date time value that can be parsed back into a {@link Date} using an appropriate
+     * {@link DateTimeFormatter}
```

Comment:
With the property behavior added to normative text above, I don't think we need this note anymore. We might want to leave something here about the convention of putting a timestamp here, and an implicit(?) requirement that it be formatted properly.